src/flamegraph/flamegraph.js

```diff
@@ -1,21 +1,63 @@
-"use strict";
-var details, searchbtn, unzoombtn, matchedtxt, svg, searching;
+"use strict"
```

Comment:
I _think_ this semicolon still needs to be there to not trip up older browsers?

Reply:
Oops, I didn't change that intentionally.